ESTIMATED_AGE_HELPERS.md

```diff
@@ -0,0 +1,287 @@
+# Age Estimation Helper Methods
```

Comment:
Do we plan to have .MD files for each new piece of functionality going forward? Or should this be removed from the SDKs and just shown on the developer documentation @Alttaf @saurabh-yoti ?